src/mame/drivers/jpmsru.cpp

```diff
@@ -909,74 +1154,128 @@ ROM_START( j_lana )
-	ROM_LOAD( "super2_2.bin", 0x0400, 0x000400, CRC(0fd5ddd0) SHA1(e8d31b009b29486d36d11052af857c609a7f1f84) )
-	ROM_LOAD( "super2_3.bin", 0x0800, 0x000400, CRC(ddd998d3) SHA1(5964da70ae4c2f174dc3d1494fc67579c221a7b7) )
+	ROM_LOAD( "goldn2.1", 0x0000, 0x400, CRC(6179002b) SHA1(a0b3311e2d2db6e88954cec681dff4ed50230826) )
+	ROM_LOAD( "goldn2.2", 0x0400, 0x400, CRC(0999d32f) SHA1(e08c852f8f3aff8ab7b73e9c0b0502ab91f9e844) )
+	ROM_LOAD( "goldn2.3", 0x0800, 0x400, CRC(d3dfd6ab) SHA1(4cf0f8977fb2c023bf2ccc8d9d74352ce32206bf) )
+	ROM_LOAD( "goldn2.4", 0x1000, 0x400, CRC(8b6922b4) SHA1(7b7fc7b0708bf96846860254fea957bcbc952923) )
 ROM_END
 
-ROM_START( j_luck2 )
+ROM_START( j_sup2p )
 	ROM_REGION( 0x10000, "maincpu", 0 )
-	ROM_LOAD( "lt_9.1", 0x0000, 0x000400, CRC(97236ce3) SHA1(f71861576f33daec3e1d371c670b535e6fd32b5e) )
-	ROM_LOAD( "lt_9.2", 0x0400, 0x000400, CRC(6e1cd083) SHA1(17edaa9880ae2a6d6d99e771e41b985527d5ed3b) )
-	ROM_LOAD( "lt_9.3", 0x0800, 0x000400, CRC(d6881e6f) SHA1(42a83f01d67a8f530ca2a10ffeff30237bdfba94) )
+	ROM_LOAD( "sup2p.1", 0x0000, 0x400, CRC(5a7349e6) SHA1(23227892ed10b4c43e987c8448053e5fa1242d39) )
+	ROM_LOAD( "sup2p.2", 0x0400, 0x400, CRC(e0eac683) SHA1(101653ab839f961b7dfd7e530418c00d4edb7ca1) )
+	ROM_LOAD( "sup2p.3", 0x0800, 0x400, CRC(14794c2b) SHA1(2f471530d524db6bab68818138760c0c22a032d5) )
 ROM_END
 
-ROM_START( j_unk )
+ROM_START( j_la )
 	ROM_REGION( 0x10000, "maincpu", 0 )
-	ROM_LOAD( "sruunk1.p1", 0x0000, 0x000400, CRC(be7d3b79) SHA1(3304dcc69e93eca2e6e89df0b18afc6874ebacf0) )
-	ROM_LOAD( "sruunk1.p2", 0x0400, 0x000400, CRC(bf19cd60) SHA1(77b0b439628589cb0db1b74a760b652519c20991) )
-	ROM_LOAD( "sruunk1.p3", 0x0800, 0x000400, CRC(25138e03) SHA1(644fc6144ea74f08dc892f106ad494ba364afe86) )
+	ROM_LOAD( "la.1", 0x0000, 0x400, CRC(21076280) SHA1(d5cf25d289f03c743f4428273ac002df3164c344) )
+	ROM_LOAD( "la.2", 0x0400, 0x400, CRC(cae10bc1) SHA1(a740946437a3b277b714f13d001783987f57bc77) )
+	ROM_LOAD( "la.3", 0x0800, 0x400, CRC(cb9362ac) SHA1(a16d43ba01b24e1b515881957c1559d33a03bcc4) )
 ROM_END
 
 #define GAME_FLAGS MACHINE_NOT_WORKING|MACHINE_MECHANICAL|MACHINE_REQUIRES_ARTWORK|MACHINE_IMPERFECT_SOUND|MACHINE_SUPPORTS_SAVE
 
 GAMEL( 1979?, j_ewn,     0,        ewn,       j_ewn,         jpmsru_state, init_jpmsru, ROT0, "JPM", "Each Way Nudger (JPM) (SRU) (revision 20, 5p Stake, £1 Jackpot)", GAME_FLAGS, layout_j_ewn )
 GAMEL( 1981?, j_ewna,    j_ewn,    ewn2,      j_ewn2,        jpmsru_state, init_jpmsru, ROT0, "JPM", "Each Way Nudger (JPM) (SRU) (revision 26A, £2 Jackpot)", GAME_FLAGS, layout_j_ewn )
 GAMEL( 1981?, j_ewnb,    j_ewn,    ewn2,      j_ewn2,        jpmsru_state, init_jpmsru, ROT0, "JPM", "Each Way Nudger (JPM) (SRU) (£2 Jackpot)", GAME_FLAGS, layout_j_ewn )
+GAMEL( 1979?, j_ewnc,    j_ewn,    ewn,       j_ewn,         jpmsru_state, init_jpmsru, ROT0, "JPM", "Each Way Nudger (JPM) (SRU) (5p Stake, £1 Jackpot", GAME_FLAGS, layout_j_ewn ) // Earlier revision
```

Comment:
The description of this one seems to be missing a closing parenthesis.

Reply:
Fixed.